x/inflation/keeper/hooks.go

```diff
@@ -19,15 +19,21 @@ func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, epochNumb
 
 	// Skip inflation if it is disabled and increment number of skipped epochs
 	if !params.EnableInflation {
-		skippedEpochs++
-		k.SetSkippedEpochs(ctx, skippedEpochs)
-		k.Logger(ctx).Debug(
-			"skipping inflation mint and distribution",
-			"height", ctx.BlockHeight(),
-			"epoch-id", epochIdentifier,
-			"epoch-number", epochNumber,
-			"skipped-epochs", skippedEpochs,
-		)
+		// check if the epochIdentifier is day before incrementing.
+		epochIdentifier := k.GetEpochIdentifier(ctx)
```

Comment:
Remove this line and move the `if` from  line 40-43 to line 19

Reply:
Okay.